test_scripts/WebEngine/WebEngineProjection/001_Registration.lua

```diff
@@ -1,42 +1,42 @@
---  Requirement summary:
---  [RegisterAppInterface] SUCCESS
---  [RegisterAppInterface] RegisterAppInterface and HMILevel
---
---  Description:
---  Check that it is possible to register App with HMI type WEB_VIEW.
---
---  Used precondition
---  SDL, HMI are running on system.
---  Mobile device is connected to SDL.
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
 --
---  Performed steps
---  appID->RegisterAppInterface(params)
+-- Description:
+-- Check that it is possible to register App with HMI type WEB_VIEW.
 --
---  Expected behavior:
---  1. SDL successfully registers application,responds to RAI request from mobile app and notifies HMI
---     SDL->HMI: OnAppRegistered(params)
---     SDL->appID: SUCCESS, success:"true":RegisterAppInterface()
---  2. SDL assignes HMILevel (NONE) to registered application and sends it a OnHMIStatus notification:
---     SDL->appID: OnHMIStatus(HMlLevel, audioStreamingState, systemContext)
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. WebEngine device is connected to SDL
+
+-- Sequence:
+-- 1. Application register with WEB_VIEW appHMIType
+--  a. SDL succesfuly registers application (resultCode SUCCESS, success:"true")
 ---------------------------------------------------------------------------------------------------
 -- [[ Required Shared Libraries ]]
 local common = require('test_scripts/WebEngine/commonWebEngine')
 
 --[[ Local Variables ]]
+local appSessionId = 1
 local appHMIType = "WEB_VIEW"
 
 --[[ General configuration parameters ]]
 config.application1.registerAppInterfaceParams.appHMIType = { appHMIType }
 config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 6
 config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 2
+config.defaultMobileAdapterType = "WS"
+common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
 
 --[[ Scenario ]]
 common.Title("Preconditions")
 common.Step("Clean environment", common.preconditions)
-common.Step("Start SDL, HMI, connect Mobile", common.start)
+common.Step("Add AppHMIType to preloaded policy table", common.updatePreloadedPT,
+  { appSessionId, appHMIType })
+common.Step("Start SDL, HMI", common.startWOdeviceConnect)
+common.Step("Connect WebEngine device", common.connectWebEngine,
+  { appSessionId, config.defaultMobileAdapterType })
```

Comment:
@ydementieiev It is enough to set `config.defaultMobileAdapterType = "WS"` that is done already.
```suggestion
common.Step("Start SDL, HMI, connect Mobile", common.start)
```

Reply:
@aderiabin thank you , done